package.json

```diff
@@ -152,6 +152,7 @@
     "color-support": "^1.1.3",
     "commander": "^9.4.1",
     "core-js": "^3.6.5",
+    "cors": "^2.8.5",
```

Comment:
`Cors` was a sub-dependency of web3@1.x that is why it worked without `cors` in package.json before. In web3@4.x we do not have `cors` as a sub-dependency.  So, I've added dependency here.